dpnp/backend/kernels/dpnp_krnl_logic.cpp

```diff
@@ -52,6 +52,28 @@ void dpnp_all_c(void* array1_in, void* result1, const size_t size)
     return;
 }
 
+template <typename _DataType, typename _ResultType>
+void dpnp_any_c(void* array1_in, void* result1, const size_t size)
+{
+    _DataType* array_in = reinterpret_cast<_DataType*>(array1_in);
```

Comment:
```suggestion
void dpnp_any_c(const void* array1_in, void* result1, const size_t size)
{
    const _DataType* array_in = reinterpret_cast<const _DataType*>(array1_in);
```
it's better have const pointer for input array.